### PR TITLE
[PR] Deleting Projects and Handling an Empty Project List

### PR DESCRIPTION
**Pull request by anjakDev**
1. Projects can be deleted one by one using the delete button in the project view.
2. If all projects are deleted/no projects are in the project list, a separate view is shown to the user. 
3. Creating new projects from an empty lists works without bugs.

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -46,10 +50,11 @@
...
 <script>
-import ProjectList from "./components/ProjectList.vue";
-import ProjectView from "./components/ProjectView.vue";
-import ListHeader from "./components/ListHeader.vue";
-import PomoView from "./components/PomoView.vue";
+import ListHeader from "@/components/ListHeader.vue";
+import ProjectList from "@/components/ProjectList.vue";
+import ProjectView from "@/components/views/ProjectView.vue";
+import PomoView from "@/components/views/PomoView.vue";
+import EmptyProjectListView from "@/components/views/EmptyProjectListView.vue";
```
Comment (anjakDev) [inline src/App.vue:57]: I just reordered the imports to group the views.

File: src/App.vue [hunk 2]
```diff
@@ -19,7 +19,10 @@
             />
           </v-col>
           <v-divider inset vertical />
-          <v-col>
+          <v-col class="d-flex align-center" v-if="!projects.length" style="height: 600px">
+            <empty-project-list-view />
+          </v-col>
+          <v-col v-if="projects.length">
```
Comment (psperber94) [inline src/App.vue:25]: I propose to use `v-else` instead.

File: src/components/views/EmptyProjectListView.vue [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<template>
+  <v-container>
+    <v-row justify="center">
+      <blockquote class="blockquote">Create a project to start tracking your time!</blockquote>
+    </v-row>
+    <v-row justify="center">
+      <img src="@/assets/undraw_stepping_up.svg" height="200px" style="opacity: 0.8" />
```
Comment (psperber94) [inline src/components/views/EmptyProjectListView.vue:7]: Love it 🥇

**Review by psperber94 — CHANGES_REQUESTED** [review]
Good stuff

**Pull request merged**